docassemble/ALToolbox/al_income.py

```diff
@@ -708,33 +805,39 @@ def _item_value_per_times_per_year(self, item, times_per_year=1):
         is_hourly = self.is_hourly and hasattr(item, "is_hourly") and item.is_hourly
 
         # If an item's value doesn't exist, use a value of 0
+        # TODO: is this behavior correct, or should it force gathering the value?
+        # What does a no-value item in the list represent?
```

Comment:
I _think_ it represented an item whose value wasn't gathered. Maybe it was an experience with a `show if` or something. Maybe it was from my lack of understanding of the context.